core/src/main/resources/org/elasticsearch/plugins/plugin-install.help

```diff
@@ -20,6 +20,8 @@ DESCRIPTION
 
     The notation of 'username/repository' refers to a github repository.
 
+    The argument can be an valid <url> which points to a download or file location for the plugin to be loaded from.
+
 EXAMPLES
 
     plugin install elasticsearch-analysis-kuromoji
```

Comment:
@rjernst @spinscale looks like if I touch that list and remove all `elasticsearch-` prefixes there, a PluginManagerTests that checks the help output vs. the `OFFICIAL_PLUGINS` list in the PluginManger breaks. I could also change that, but I'm not sure if I'm going down a rabbit hole with this. I think this should to be done in a consistent way in an additional PR.